Index: src/main/java/Model/CSVReader.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Model;\r\n\r\npublic class CSVReader {\r\n  \r\n  \r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Model/CSVReader.java b/src/main/java/Model/CSVReader.java
--- a/src/main/java/Model/CSVReader.java	(revision 29d0031b7636faf23896935650e5c5c28215faed)
+++ b/src/main/java/Model/CSVReader.java	(date 1644332363171)
@@ -1,6 +1,8 @@
 package Model;
 
 public class CSVReader {
+
+
   
   
 }
Index: src/main/java/Model/EmployeeRecord.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Model/EmployeeRecord.java b/src/main/java/Model/EmployeeRecord.java
new file mode 100644
--- /dev/null	(date 1644332651023)
+++ b/src/main/java/Model/EmployeeRecord.java	(date 1644332651023)
@@ -0,0 +1,5 @@
+package Model;
+
+public class EmployeeRecord {
+
+}
